app/src/main/java/com/android/sample/ui/preferences/PreferencesScreen.kt

```diff
@@ -42,9 +46,24 @@ fun PreferencesScreen(
   Scaffold(
       topBar = {
         TopAppBar(
-            title = { Text("Modify preferences", fontSize = 20.sp, fontWeight = FontWeight.Bold) },
-            navigationIcon = { ArrowBack(navigationActions) },
-            colors = TopAppBarDefaults.smallTopAppBarColors(containerColor = Color.White),
+            title = {
+              Text(
+                  "Modify preferences",
+                  fontSize = 20.sp,
+                  fontWeight = FontWeight.Bold,
+                  color = Color.White)
+            },
+            navigationIcon = {
+              IconButton(
+                  onClick = { navigationActions.goBack() },
+                  modifier = Modifier.testTag("ArrowBackButton")) {
+                    Icon(
+                        imageVector = Icons.Outlined.ArrowBack,
+                        contentDescription = "Back",
+                        tint = Color.White)
+                  }
+            },
+            colors = TopAppBarDefaults.smallTopAppBarColors(containerColor = DarkBlue),
             modifier = Modifier.testTag("preferencesTopBar"))
       },
```

Comment:
Instead of repeating the TopAppBar in both SettingsScreen and PreferencesScreen, you should maybe consider creating a reusable component to centralize styling and reduce redundancy in these two files